planner/core/memtable_predicate_extractor_test.go

```diff
@@ -1420,3 +1420,126 @@ func (s *extractorSuite) TestTiDBHotRegionsHistoryTableExtractor(c *C) {
 		}
 	}
 }
+
+func (s *extractorSuite) TestTikvRegionPeersExtractor(c *C) {
+	se, err := session.CreateSession4Test(s.store)
+	c.Assert(err, IsNil)
+
+	var cases = []struct {
```

Comment:
What about adding a test case that contains other columns to make sure we can only get the `region_id` and `store_id`?